test/release.test.js

```diff
@@ -159,11 +167,7 @@ tap.test(
       stubs.coreStub.setFailed,
       `Something went wrong while deleting the release. \n Errors: Something went wrong in the release`
     )
-    sinon.assert.neverCalledWith(stubs.runSpawnStub, 'npm', [
-      'publish',
-      '--tag',
-      'latest',
-    ])
+    sinon.assert.notCalled(stubs.publishToNpmStub)
```

Comment:
The previous assert was not "strong" enough, it would always succeed after the introduction of the `publishToNpm` function in a previous PR.